ui/src/image_annotator_polygon.ts

```diff
@@ -1,41 +1,113 @@
 import { F, S, U } from "h2o-wave"
-import { DrawnPoint, DrawnShape, ImageAnnotatorPoint, Position } from "./image_annotator"
+import { DrawnPoint, DrawnShape, ImageAnnotatorPoint, ImageAnnotatorRect, Position } from "./image_annotator"
 import { ARC_RADIUS } from "./image_annotator_rect"
 
 export class PolygonAnnotator {
   private currPolygonPoints: ImageAnnotatorPoint[] = []
+  private boundaryRect: ImageAnnotatorRect | undefined
```

Comment:
To match the type of 
```ts
export interface ImageAnnotatorPolygon {
  /** List of polygon points. */
  vertices: ImageAnnotatorPoint[]
  /** The polygon's boundary rectangle. */
  boundaryRect?: ImageAnnotatorRect
}
```
but it is not relevant anymore since the prop should not be public facing and needs to be removed.